src/test/java/com/missafternoon/budjen/transactions/JSONInterpreterTest.java

```diff
@@ -0,0 +1,63 @@
+package com.missafternoon.budjen.transactions;
+
+import com.missafternoon.budjen.formatting.DateConverter;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class JSONInterpreterTest {
+
+    private JSONInterpreter target = new JSONInterpreter();
+    private TestTransactionData testData = new TestTransactionData();
+
+    @Test
+
+    public void writeJavaObjectIntoJSONString() {
+        String json = target.convertToString(testData.getTransactionData());
```

Comment:
This throws an unhelpful message - I know!
The issue is that the getters and setters on the Transaction class have to be `public`.

... obvious right? 🙄